docs/README.md

```diff
@@ -1920,60 +1914,277 @@ mutation {
 }
 ```
 
-> The following message appears in the socket
-
+2. The following message (after formatting it) appears through the socket connection we had opened:
 ```json
 {
-  "data": {
-    "CartReadModel": {
-      "id": "demo",
-      "items": [
-        {
-          "sku": "ABC_01",
-          "quantity": 2
-        },
-        {
-          "sku": "ABC_02",
-          "quantity": 3
-        }
-      ]
+  "id": "1",
+  "type": "data",
+  "payload": {
+    "data": {
+      "CartReadModel": {
+        "id": "demo",
+        "items": [
+          {
+            "sku": "ABC_01",
+            "quantity": 2
+          },
+          {
+            "sku": "ABC_02",
+            "quantity": 3
+          }
+        ]
+      }
     }
   }
 }
 ```
 
-It is also possible to use a GraphQL client to subscribe to changes in the application. However, it might be necessary to include an access token, provided when the user signs-in, in the request if the operation requires authorization. This token should be added to the GraphQL client as an operation option through a middleware. An example can be seen below:
+>**Note**: Remember that, in case you want to subscribe to a read model that is restricted to a specific set of roles, you must send the **access token** retrieved upon sign-in. Check ["Authorizing operations"](#authorizing-operations) to know how to do this.
+
+#### Using Apollo Client
+One of the best clients to connect to a GraphQL API is the [Apollo](https://www.apollographql.com/) client.
+There will probably be a version for your client technology of choice. These are the main ones:
+- [For Javascript/Typescript](https://www.apollographql.com/docs/react/) ([Github](https://github.com/apollographql/apollo-client))
+- [For iOS](https://www.apollographql.com/docs/ios/) ([Github)](https://github.com/apollographql/apollo-ios))
+- [For Java/Kotlin/Android](https://www.apollographql.com/docs/android/) ([Github](https://github.com/apollographql/apollo-android))
+
+We recommend referring to the documentation of those clients to know how to 
+use them. Here is an example of how to fully instantiate the Javascript client so that it works for queries, mutations and subscriptions:
+```typescript
```

Comment:
If all this code is available in the apollo client website I would remove it from here, what do you think?

Reply:
It is not exactly the same code, and it is scattered over some sections. If you follow along, you'll see that we will be improving this code to add tokens and the ability to refresh tokens, things that are not int he Apollo website.

Do you think it is not a good idea to put this code in the documentation?